File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/SettingsSearch.java
package com.varun.scholar.business.data.entities1;

import javax.persistence.Entity;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class SettingsSearch extends AbstractEntity {
	int institutionDetailsId;
	String name;
	String address;
	String phone;
	String fax;
}

File: scholar-web/src/main/webapp/app/model/administration/settings/general/SettingsSearch.js
Ext.define('scholar.model.administration.settings.general.SettingsSearch', {
	extend : 'Ext.data.Model',
	idProperty: 'id',
	fields : [ {
		name : 'id',
		type : 'int'
	}, {
		name : 'name',
		type : 'string'
	},
	{
		name : 'address',
		type : 'string'
	},{
		name : 'phone',
		type : 'string'
	},
	{
		name : 'fax',
		type : 'string'
	}]

});